.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+repos:
+  - repo: https://github.com/timothycrosley/isort
+    # isort config is in setup.cfg
+    rev: 5.6.4
```

Comment:
We still use version 4.2.5 which isn't compatible with isort 5.x

Reply:
updated to match.